CRM/Contribute/Form/Contribution/Main.php

```diff
@@ -439,12 +439,7 @@ public function buildQuickForm() {
     }
     if ($this->_pcpId && empty($this->_ccid)) {
       if ($pcpSupporter = CRM_PCP_BAO_PCP::displayName($this->_pcpId)) {
-        $pcp_supporter_text = ts('This contribution is being made thanks to the effort of <strong>%1</strong>, who supports our campaign.', [1 => $pcpSupporter]);
-        // Only tell people that can also create a PCP if the contribution page has a non-empty value in the "Create Personal Campaign Page link" field.
-        $text = CRM_PCP_BAO_PCP::getPcpBlockStatus($this->_id, 'contribute');
```

Comment:
You've actually made a change here - you are passing in $this->_pcpId but then using it in place of $this->_id

Reply:
@eileenmcnaughton Thanks for spotting - I've now added the contributionPageID as a parameter so nothing is being changed anymore.